core/dbt/compilation.py

```diff
@@ -107,6 +107,18 @@ def _extend_prepended_ctes(prepended_ctes, new_prepended_ctes):
         _add_prepended_cte(prepended_ctes, new_cte)
 
 
+def _get_tests_for_node(manifest: Manifest, unique_id: UniqueID) -> List[UniqueID]:
+    """ Get a list of tests that depend on the node with the
+    provided unique id """
+
+    return [
+        node.unique_id
+        for _, node in manifest.nodes.items()
+        if node.resource_type == NodeType.Test and
```

Comment:
I don't understand Python syntax but does the `if` not need to be indented under the `for` loop?

Reply:
Nope, not here 😎 This is just one long line using [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) that I broke up into multiple lines! It is almost equivalent to:
```python
result_list = []
for _, node in manifest.nodes.items():
    if node.resource_type == NodeType.Test and unique_id in node.depends_on_nodes:
        result_list.append(node.unique_id)
return result_list
```